test/uk/gov/hmrc/vatsubscription/models/updateVatSubscription/request/RequestedChangesSpec.scala

```diff
@@ -52,5 +52,19 @@ class RequestedChangesSpec extends UnitSpec {
 
       RequestedChanges.latestDESApi1365Writes.writes(model) shouldBe result
     }
+
+    "output a correctly formatted RequestedChanges json for the latest DES API1365 writes with email" in {
```

Comment:
I don't think this test is needed as the test case above is identical apart from the test name.